examples/src/components/BarChartExample.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import { AfmComponents } from '@gooddata/react-components';
+
+import '@gooddata/react-components/styles/css/main.css';
+
+export class BarChartExample extends Component {
+    onLoadingChanged(...params) {
+        // eslint-disable-next-line no-console
+        console.info('onLoadingChanged', ...params);
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        console.info('onLoadingChanged', ...params);
+    }
+
+    render() {
+        const afm = {
+            measures: [
+                {
+                    localIdentifier: 'amount',
+                    definition: {
+                        measure: {
+                            item: {
+                                uri: '/gdc/md/la84vcyhrq8jwbu4wpipw66q2sqeb923/obj/336278'
+                            },
+                            aggregation: 'sum'
+                        }
+                    },
+                    alias: 'Sum of _Close [BOP]',
+                    format: '#,##0'
+                },
+                {
+                    localIdentifier: 'amount2',
+                    definition: {
+                        measure: {
+                            item: {
+                                uri: '/gdc/md/la84vcyhrq8jwbu4wpipw66q2sqeb923/obj/1536'
+                            },
+                            aggregation: 'sum'
+                        }
+                    },
+                    alias: 'Sum of Amount',
+                    format: '#,##0'
+                }
+            ],
+            attributes: [
+                {
+                    displayForm: {
+                        uri: '/gdc/md/la84vcyhrq8jwbu4wpipw66q2sqeb923/obj/355'
+                    },
+                    localIdentifier: 'year'
+                }
+            ]
+        };
+
+        return (
+            <div style={{ height: 300 }} >
+                <AfmComponents.BarChart
```

Comment:
I don't like this syntax... I think it would be better to spred it before the call and then just call
```
const { BarChart } = AfmComponents;
 <BarChart ... />
```